lib/grab-module-data.js

```diff
@@ -48,7 +48,9 @@ function grabModuleData(context) {
         return;
       }
 
-      if (data === '') {
+      // Prior to npm 7, returns empty string for invalid request
+      // In npm 7 newline is returned for invalid request
```

Comment:
this isn't something npm 7 will fix? it's likely to break a number of automated tools

Reply:
We can follow up on the... but this edge case is specifically when you call something like `npm view --json lodash@0.0.0. CITGM was testing an error case and was reliant on the STDOUT of that error case. I'm skeptical that many production tools are relying on this behavior.

Reply:
that the stdout is falsy in an error case? i'm pretty sure i have one that relies on that, but i'd have to dig it up.

Reply:
I'll dig in next week and see about getting this patched, it honestly shouldn't be that hard a change to make I just don't know where to poke the codebase :P